Posts/2021/04/tfl-IsUserALocalAdministrator.md

```diff
@@ -0,0 +1,130 @@
+---
+post_title: Is a User A Local Administrator
+username: tfl@psp.co.uk
+Catagories: PowerShell
+tags: local users, security, logon scripts
+Summary: In a logon script, how you can tell if the user is a local administrator
+---
+
+**Q:** Some of the things we do in our logon scripts require the user to be a local administrator. How can the script tell if the user is a local administrator or not, using PowerShell 7.
```

Comment:
Technically this module is not part of PowerShell 7. It is a Windows PowerShell 5.1 module. But you can use it in PowerShell 7. PS7 automatically loads it through the WindowsCompatibility feature.

Reply:
Actually it doesn't . Just tested it and it loads natively. :-)

But I have redrafted the text in the next commit.